### PR TITLE
fix cuda graphs for qwen2-vl

### PR DESCRIPTION
**Pull request by drbh**
The PR: adds support for multidimensional position ids on batch to enable cuda graphs on qwen2-vl. This PR is a fast followup to the original support of qwen2-vl: https://github.com/huggingface/text-generation-inference/pull/2689

### CODE REVIEW

**Comment by drbh** [comment]
thank you again @sywangyi for providing the changes that fix the `lm_head_indices` and output the correct logits 🙏

ref: [fix qwen2 failure in intel cpu](https://github.com/huggingface/text-generation-inference/pull/2708/commits/17de5998e5b34dbc863d84fb5285d3692b02c667)

**Review by Narsil — APPROVED** [review]
